code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -35,6 +35,7 @@
 	var/obj/item/slimepotion/slime/current_potion
 	var/max_slimes = 5
 	var/monkeys = 0
+	var/static/list/already_researched = list() //List of cores that've already been used for research
```

Comment:
I'm not a fan of this being saved in a static list on the xenobio camera, can we not use the research subsystem for this?

Reply:
Possibly, but this is the only place it's ever going to be referenced. It makes more sense to just tie it to the cameras IMO.

Reply:
The name is a bit generic for a static list, so if you're keeping it there i'd make it more specific.
I do second the opinion that it'd belong to the research subsystem though.